src/generated/sdk.ts

```diff
@@ -318,7 +318,7 @@ export type Entry = {
 export type Equipment = {
   __typename?: 'Equipment';
   id: Scalars['ID'];
-  owner: Maybe<User>;
+  owner: User;
```

Comment:
I think your local scheduler images are out-of-date and that's why the pipeline fails.